ci/docker-compose.yml

```diff
@@ -89,7 +89,7 @@ services:
       KUDU_MASTER: "false"
 
   omniscidb:
-    image: omnisci/core-os-cpu:v5.1.0
+    image: omnisci/core-os-cpu-dev:latest
```

Comment:
@xmnlab is this right? why are we not pinning exactly (to 5.2?)

Reply:
My several pull requests aren't working with the latest omnisci release due to a lack of some changes.

Reply:
this was discussed on https://github.com/ibis-project/ibis/pull/2084